package.json

```diff
@@ -2,8 +2,21 @@
   "name": "healthz-prober",
   "version": "1.0.0",
   "description": "probes healthz endpoints and reports to slack",
-  "main": "index.ts",
+  "main": "src/index.ts",
   "scripts": {
+    "start": "npm run build && node --env-file=.env dist/index.js",
+    "start:watch": "node --env-file=.env --watch dist/index.js",
+    "build": "tsc",
+    "build:watch": "tsc -w",
+    "dev": "(start npm run build:watch || npm run build:watch) & npm run start:watch",
```

Comment:
Running dev mode needed a different approach on windows: doesn't show the output.

Reply:
Thanks for the catch! Applied in [5ad47d8](https://github.com/exxocism/node-healthz-prober/pull/1/commits/5ad47d8519d075191a582896215c3237e0ba83ca)

Reply:
Still got a issue with macOS. it lingers and doesn't comply to ctrl + key. 🤔

Reply:
I think we can handle this in an another ticket. How about you?
Since it's at least running, we can manage it as a known issue.

Reply:
As we talked in slack, We are managing it on https://exxocism.atlassian.net/browse/NHP-6